cinch/group_vars/fedora

```diff
@@ -3,17 +3,14 @@ repositories:
     mirrorlist: "{{ fedora_mirrors }}repo=rawhide"
 
 _download_repositories:
-  - http://pkg.jenkins-ci.org/redhat-stable/jenkins.repo
+  - https://fedorapeople.org/~semyers/jenkins-rpm/jenkins1651.repo
```

Comment:
At this point I think we should default to Jenkins 2.60.3 for new deployments.

Reply:
That repo contains both 1.651.3 and 2.60.3. I only made this change because it is a much faster mirror for people who still want the older package and it was taking well over an hour to install from the upstream repo on Fedora.

Reply:
Understood.  Certainly there are other places where we would need to change to default to 2.60.3.   We should do that eventually, but perhaps not in this PR.

Reply:
There's a variable in the jenkins_master role's defaults: jenkins_version, I believe. That is what would need to switch. I'd like us to get to 2.60.3 before we call ourselves 1.0, for sure. Shall we open a ticket for that?